tests/foreman/cli/test_domain.py

```diff
@@ -184,6 +190,34 @@ def test_negative_create(self):
                 with self.assertRaises(CLIFactoryError):
                     make_domain(options)
 
+    @tier2
+    @skip_if_bug_open('bugzilla', 1398392)
+    @run_only_on('sat')
+    def test_negative_create_with_invalid_dns_id(self):
+        """Attempt to register a domain with invalid id
+
+        @id: 4aa52167-368a-41ad-87b7-41d468ad41a8
+
+        @expectedresults: Error is raised and user friendly message returned
+
+        @BZ: 1398392
+
+        :CaseLevel: Integration
+        """
+        with self.assertRaises(CLIFactoryError) as context:
+            make_domain({
+                'name': gen_string('alpha'),
+                'dns-id': -1,
+            })
+        valid_messages = ['Invalid smart-proxy id', 'Invalid capsule id']
+        exception_string = str(context.exception)
+        messages = [
+            message
+            for message in valid_messages
+            if message in exception_string
+        ]
+        self.assertGreater(len(messages), 0)
```

Comment:
Ohh, now I see
```
resolution show message "Invalid smart-proxy id"
nightly show message "Invalid capsule id"
```
So, in nightly there is one message and in 6.2.x is another. Then we need to submit in master and 6.2.z branches code with different messages